packages/vscode-extension/src/commonlib/codeFlowLogin.ts

```diff
@@ -61,6 +65,11 @@ export class CodeFlowLogin {
     this.msalTokenCache = this.pca.getTokenCache();
     this.accountName = accountName;
     this.status = loggedOut;
+    this.isCodeSpace = false;
+  }
+
+  public updateIsCodeSpace(isCodeSpace: boolean) {
+    this.isCodeSpace = isCodeSpace;
   }
```

Comment:
Do we need this set API? Whether the env is in codespaces can be detected from environment variable `codespaces`.
See https://docs.github.com/en/codespaces/developing-in-codespaces/default-environment-variables-for-your-codespace for more details.

Reply:
Done.